### DISCUSSION TITLE
Welcome to rocm-blogs Discussions!

### DISCUSSION THREAD

**Discussion by saadrahim** [ROCm/rocm-blogs#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kyleL-lkf** [comment]
![bc391de7-383e-47cc-9321-5cde23e003b7](https://github.com/user-attachments/assets/92edff05-17b1-4f66-969e-2cd3440872ab)
检测不到GPU是为什莫

**Comment by lucasdavidglacko-beep** [comment]
The ecosystem has grown significantly for AI and GPU computing workloads, and better optimization guides will make adoption even smoother. I also tested a [chipotle macro calculator ](https://chipotlecalorie.com/)recently, and it showed how important accurate data tracking and clean performance metrics are in every type of project.
